source/switching_to_live/set_up_a_live_smartpay_account/index.html.md.erb

```diff
@@ -1,28 +1,24 @@
 ---
-title: Switching to live
-weight: 100
+title: Connect your live account to SmartPay
+weight: 400
 ---
 
-# Set up a live Smartpay account
+# Connect your live account to SmartPay
 
-Please read the guidance on what to do [before you switch to
-live](/switching_to_live/before_you_switch_to_live/#before-you-switch-to-live)
-first.
+Before you start, you must:
 
-You should follow these instructions in order.
+- read about [what to do first](/switching_to_live/#before-you-start)
+- [request a live GOV.UK Pay account](/switching_to_live/#1-request-a-live-gov-uk-pay-account)
 
-You can also read about setting up live
-[ePDQ](/switching_to_live/set_up_a_live_epdq_account) and
-[Worldpay](/switching_to_live/set_up_a_live_worldpay_account) accounts.
+You must also be an admin on your GOV.UK Pay account.
 
-## Configure your user profile on Smartpay
+## Configure your user profile on SmartPay
 
-Sign in to Smartpay and select the correct merchant account. You can select
-the organisation name at the top of the page to view all merchant accounts.
+Sign in to your [SmartPay account](https://ca-live.barclaycardSmartPay.com/ca/ca/login.shtml). Select your organisation name at the top of the page, then the correct merchant account.
```

Comment:
The bit about selecting the correct organisation name and merchant account part applies to the entire page but it’s only referenced within this one section.

Reply:
I had a look at this and I think it's ok? If I remember right, once you've selected it once, you don't need to select it again. Let me know if I'm wrong though.

Reply:
I suppose I was thinking of if someone came to this page and decided to jump straight to the notification settings bit for whatever reason. If you think that’s unlikely, happy to keep it as it is.

Reply:
I see what you mean - but I think we'd need bigger changes if this page needed to work for users jumping into the sections. At the mo we're basing it on a user need of 'doing all the live settings in one go', so on that basis I think we'll be good. But let's keep an eye on any related support tickets after we publish.